main/options/options.cc

```diff
@@ -946,6 +948,7 @@ void readOptions(Options &opts,
         opts.lspDocumentHighlightEnabled =
             enableAllLSPFeatures || raw["enable-experimental-lsp-document-highlight"].as<bool>();
         opts.lspSignatureHelpEnabled = enableAllLSPFeatures || raw["enable-experimental-lsp-signature-help"].as<bool>();
+        opts.lspMultipleDirEnabled = enableAllLSPFeatures || raw["enable-experimental-lsp-multiple-dir"].as<bool>();
```

Comment:
Would you mind only enabling this if it's explicitly enabled? I know it goes a bit against the `experimental` part of `enable-all-experimental-lsp-features`, but the features we have in that set are currently fairly stable, and I think this new feature could use some stabilization before we enable it with the rest.

```suggestion
        opts.lspMultipleDirEnabled = raw["enable-experimental-lsp-multiple-dir"].as<bool>();
```